userguide/kaui/includes/custom-fields.adoc

```diff
@@ -0,0 +1,106 @@
+
+image::../assets/img/kaui/CustomFields-More.png[width=850,align="center"]
+
+[TIP]
+*Tip:* On the Custom Fields page, you can open the object (account, invoice, etc.) by clicking the link in the Object ID column.
+
+=== View Custom Fields for an Account
+
+This section explains how to view any custom fields for a specific account or other objects associated with that account (e.g., invoices, payments, subscriptions).
+
+. Open the account on the Account page.
+. Click *Custom Fields* on the sub-menu.
+
+Kaui opens the Custom Fields page and displays only the custom fields assigned to that account or its related objects:
+
+image::../assets/img/kaui/CustomFields-Specific.png[width=850,align="center"]
+
+=== Create and Attach a Custom Field
+
+To create a custom field and attach it to an object:
+
+. Open the Custom Fields page as explain in "<<View All Custom Fields>>".
++
+image::../assets/img/kaui/CustomFields-Populated.png[width=850,align="center"]
++
+. Click the plus sign (image:C:\_My Documents\FlowWritingLLC\Projects\Kill Bill\Documentation\killbill-docs\userguide\assets\img\kaui\i_PlusGreen.png[] ) next to *Custom Fields.*
++
+Kaui displays the Add New Custom Field screen:
++
+image::../assets/img/kaui/AddNewCustomField.png[width=350,align="center"]
++
+. Fill in the fields. For field descriptions, see "<<Custom Field Descriptions>>".
++
+. Click the *Save* button. Kaui displays the new custom field on the Custom Fields page.
+
+image::../assets/img/kaui/CustomFields-More.png[width=850,align="center"]
+
+=== Custom Field Descriptions
+
+[cols="1,3"]
+|===
+^|Field ^|Description
+
+|Object ID
+|The identifier of the object to which you want to attach a custom field.
+
+*Note:* To copy the identifier for the object, open a separate browser tab and copy the ID from the URL displayed on your browser's address line:
+
+ _Example:_
+
+URL: `\https://demo.killbill.io/accounts/1033706a-07de-4ed3-b0e0-0f1d09639b90/invoices/421d16de-6d19-41a0-b9e5-781ab3eedf1c`
+
+Invoice ID: `421d16de-6d19-41a0-b9e5-781ab3eedf1c`
+
+|Object type
+|The type of object that the custom field is being attached to, such as an account or payment.
+
+|Name
+|The name of the custom field (_Example:_ Level).
+
+|Value
+|The value associated with this custom field (_Example:_ Medium).
+
+|===
+
+=== Delete a Custom Field
+
+[WARNING]
+*Warning:* Kaui does not ask you to confirm your deletion; use this feature with caution.
+
+In Kaui, you can delete a custom field from the current tenant only if it hasn't been attached to an object. If you try to delete such a custom field, Kaui will prevent you.
+
+To delete a custom field:
+
+. At the top of the screen, click the tag icon and select *Custom Fields*.
++
++image::../assets/img/kaui/TagDropdown-CustomFields.png[width=150,align="center"]
++
+Kaui displays the Custom Fields page:
++
+image::../assets/img/kaui/CustomFields-Populated.png[width=850,align="center"]
++
+. On the custom field row, click *Destroy*.
```

Comment:
Is this correct? I do not see a **Destroy** link in the custom field row.

Reply:
@reshmabidikar - Stéphane said that functionality appears to be missing from Kaui. You don't see "Destroy" when you're logged in, do you?

Reply:
I have removed the "deleting" section from the Custom Fields chapter.